entries/w/wefunder.com.json

```diff
@@ -2,9 +2,11 @@
     "Wefunder": {
         "domain": "wefunder.com",
         "tfa": [
+            "totp",
             "u2f"
         ],
         "documentation": "https://help.wefunder.com/en_US/multifactor-authentication",
```

Comment:
Documentation links cannot require sign-in. Could you provide a screenshot of the page or link to a page that isn't password protected?